generated/nifgen/nifgen/_grpc.py

```diff
@@ -0,0 +1,514 @@
+# -*- coding: utf-8 -*-
```

Comment:
this file needs to be renamed to match the name of the class per nimi-python's convention.

Reply:
This is a generated file, of course, and Tom just needs to merge in the latest commits and re-codegen.